docs/networks/mainnet/lukso-mainnet.md

```diff
@@ -0,0 +1,228 @@
+
+It's a big responsibility, and not a simple set-it-and-forget-it kind of job; you need to care for your node for as long as it's staking.
+
+## A Proof of Stake Chain
+
+<--! TODO: Hugo/Fabian, please review this text and define the LYX quantity to be deposited to run a node , or point to the community/governance discussion about this amount-->
+
+Staking LYX in Proof of Stake is done via validators. A validator is essentially a single [Beacon Chain](https://ethereum.org/en/upgrades/beacon-chain/) address to which ~~100 LYX~~ is deposited on the Execution layer. Validators are responsible for maintaining the consistency and security of the Beacon Chain. They do this by listening for transactions and new block proposals and attesting that the proposed block contains legal, valid transactions by doing some number crunching and verification behind the scenes. Occasionally, they get to propose new blocks themselves.
+
+Validators are assigned attestations and block proposals on a randomized schedule. This is very different from the old Proof of Work system, where everyone was constantly trying to race each other and come up with the next block before everyone else. This means that unlike Proof of Work where miners weren't guaranteed to earn a block reward unless they found the next block, Proof of Stake validators are guaranteed to have slow, steady income as long as they perform their duties. If a validator is offline and misses an attestation or a block proposal, it will be slightly penalized. The penalties are quite small though; as a rule of thumb, if a validator is offline for X hours, it will make all of its lost LYX back after the same X hours of being back online.
+
+## LUKSO's Node Requirements
+
+Here is what you will need to run a Lukso node :
+
+- A stable Internet connection. The longer you stay online, the better your rewards. A spotty Internet connection will hurt your returns, and by extension, the LYX ratio growth.
+- At least 10Mbps of bandwidth both up and down. A LUKSO node usually takes around 8Mbps to 10Mbps up & down of network traffic, depending on your configuration and number of nodes you decide to run.
+- No data cap imposed by your ISP. It is not suggest to run a node if your Internet plan comes with a monthly data cap.
+- Stable electricity. For the same reason as needing a stable Internet connection, you also want to have reliable power. This can be mitigated with a large UPS (backup battery) to deal with short blackouts.
+- A computer or Virtual Machine with sufficient specs. The computer can be a local machine, or it can be a Virtual Private Server (VPS) hosted in the cloud.
+
+The following specifications are considered minimum requirements:
+
+Linux or macOS Operating System
+
+- Quad core CPU (or dual-core hyperthreaded); both `x64` and `arm64` are supported
+- 8 GB of RAM (preferably DDR4); Typical configurations tend to use 16 or 32 GB of RAM
+- 2 TB of free SSD Disk Space
```

Comment:
I think 2tb is more than enough bc the lukso chain won't be big. I think even 1tb is enough. (May not be future proof but it should work for a couple of months i think)